libraries/AP_RangeFinder/AP_RangeFinder_PWM.cpp

```diff
@@ -67,7 +67,7 @@ bool AP_RangeFinder_PWM::get_reading(uint16_t &reading_cm)
     if (value_us == 0 || sample_count == 0) {
         return false;
     }
-    reading_cm = (value_us/sample_count) * 1e-1f; // correct for LidarLite.  Parameter needed?
+    reading_cm = value_us/(sample_count * 10.0f); // correct for LidarLite.  Parameter needed?  Converts from m -> cm here
```

Comment:
No `.0f`!

Reply:
I actually like the 10.0f.

Reply:
The problem is this is actually pure integer math into a return of integer, promoting to float in the middle and then implicit cast to int is wasting time. The whole bug here is that this expression should simply never be a float.

Reply:
/me checks type of reading_cm

Whoops.

Fixed.